cmd/forwarder/http.go

```diff
@@ -131,6 +131,12 @@ func (s *httpServer) Run() error {
 			s.pAuthSuccesses.Inc(1)
```

Comment:
While you are in here ... this else should be removed and dedent the `s.pAuthSeuccesses.Inc(1)`